[dynamo] Recursively realize the stack_values

anijain2305 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #152883
* __->__ #152853

Might also fix - https://github.com/pytorch/pytorch/issues/135696

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

anijain2305 committed:
[dynamo] Recursively realize the stack_values

[ghstack-poisoned]

anijain2305 committed:
Update on "[dynamo] Recursively realize the stack_values"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Lucaskabela reviewed (approved):
LGTM; thanks for cleaning up the test as well!

anijain2305 commented:
@pytorchbot merge

anijain2305 committed:
Update on "[dynamo] Recursively realize the stack_values"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

anijain2305 committed:
Update on "[dynamo] Recursively realize the stack_values"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

anijain2305 commented:
@pytorchbot merge -i

anijain2305 committed:
Update on "[dynamo] Recursively realize the stack_values"


Might also fix - https://github.com/pytorch/pytorch/issues/135696

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

anijain2305 committed:
Update on "[dynamo] Recursively realize the stack_values"


Might also fix - https://github.com/pytorch/pytorch/issues/135696

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

anijain2305 commented:
@pytorchbot merge

anijain2305 commented:
@pytorchbot merge -i
